### PR TITLE
remove breakpoints on tab close

### PR DESCRIPTION
**Pull request by pahans**

### CODE REVIEW

File: modules/web/js/debugger/debug-manager.js [hunk 1]
```diff
@@ -137,13 +137,26 @@ define(['require', 'jquery', 'backbone', 'lodash', 'event_channel', './channel',
         return this.enable;
     };
 
-    DebugManager.prototype.getDebugPoints = function () {
-        return this.debugPoints;
+    DebugManager.prototype.getDebugPoints = function (fileName) {
```
Comment (hemikak) [inline modules/web/js/debugger/debug-manager.js]: Can we have doc comments.

File: modules/web/js/debugger/debug-manager.js [hunk 2]
```diff
@@ -137,13 +137,26 @@ define(['require', 'jquery', 'backbone', 'lodash', 'event_channel', './channel',
...
+        var breakpoints = _.map(this.debugPoints, function(breakpoint) {
+            if(breakpoint.fileName === fileName)  {
+                return breakpoint.lineNumber;
+            }
+        });
+        return breakpoints;
     };
 
+    DebugManager.prototype.removeAllBreakpoints = function(fileName) {
```
Comment (hemikak) [inline modules/web/js/debugger/debug-manager.js]: Can we have doc comments.

File: modules/web/js/debugger/debug-manager.js (hunk 1 above)
Comment (pahans) [inline modules/web/js/debugger/debug-manager.js]: added doc comments in this PR https://github.com/ballerinalang/composer/pull/1996

File: modules/web/js/debugger/debug-manager.js (hunk 2 above)
Comment (pahans) [inline modules/web/js/debugger/debug-manager.js]: added doc comments in this PR https://github.com/ballerinalang/composer/pull/1996

**Pull request merged**